### PR TITLE
Removing unused variables

### PR DESCRIPTION
**Pull request by shimwell**
Found a few unused variables in the code.

Perhaps best to remove them

### CODE REVIEW

File: src/geouned/GEOReverse/Modules/MCNPinput.py [hunk 1]
```diff
@@ -1077,7 +1077,6 @@ def get_hyperboloid_parameters(eVal, eVect, T, k, iaxis):
     elliposoid = False
     if iaxis is None:
         iaxis = np.argmin(np.abs(eVal))
-        ellipsoid = True
```
Comment (psauvan) [inline src/geouned/GEOReverse/Modules/MCNPinput.py:1080]: This should variable not be removed. In fact there is a typo in the spelling of the variable in line 1077 "elliposoid" which is the variable used in line 1144.
Please keep the variable in line 1080 and rename the variable in lines 1077 and 1144

**Review by psauvan — CHANGES_REQUESTED** [review]
Please make the change suggested in file 
src/geouned/GEOReverse/Modules/MCNPinput.py

File: src/geouned/GEOReverse/Modules/MCNPinput.py (hunk 1 above)
Comment (shimwell) [reply src/geouned/GEOReverse/Modules/MCNPinput.py:1080]: excellent spot @psauvan

Comment (shimwell) [reply src/geouned/GEOReverse/Modules/MCNPinput.py:1080]: I've corrected the spelling typos and put ellipsoid back as requested.

This PR now has slightly different behaviour compared to the dev branch, as it fixes an undiscovered bug. Perhaps this helps with some of the geometries that lose particles during transport

**Review by psauvan — APPROVED** [review]

**Pull request merged**